test/http2.test.js

```diff
@@ -3,17 +3,33 @@
 const { test } = require('tap')
 const Fastify = require('fastify')
 const eh = require('../index')
+const http2 = require('http2')
 
-test('Should throw when http2 server', async (t) => {
-  t.plan(1)
+test('initial test', (t) => {
+  t.plan(3)
   const fastify = Fastify({ http2: true })
   fastify.register(eh)
   fastify.get('/', async (request, reply) => {
     await reply.writeEarlyHintsLinks([
-      'Link: </style.css>; rel=preload; as=style',
-      'Link: </script.js>; rel=preload; as=script'
+      '</style.css>; rel=preload; as=style',
```

Comment:
here I removed the "Link:"-prefix but in http1 we expect them to be prefixed with "Link:"